src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -93,86 +93,62 @@ public String getDeploymentId () {
-                    failJob("Failure encountered while downloading transit bundle.");
+            // Wait for otp-runner to produce first status file
+            long statusCheckStartTime = System.currentTimeMillis();
+            String statusUrl = String.join("/", ipUrl, OTP_RUNNER_STATUS_FILE);
+            boolean otpRunnerStatusAvailable = false;
+            while (!otpRunnerStatusAvailable) {
+                // If the request is successful, the OTP instance has started.
+                waitAndCheckInstanceHealth("otp-runner status file availability check: " + statusUrl);
+                otpRunnerStatusAvailable = checkForSuccessfulRequest(statusUrl);
+                long maxOtpRunnerStartupTimeMillis = 5 * 60 * 1000;
+                if (taskHasTimedOut(statusCheckStartTime, maxOtpRunnerStartupTimeMillis)) {
+                    failJob("Job timed out while waiting for otp-runner to produce a status file!");
                     return;
                 }
-                long bundleDownloadSeconds = (System.currentTimeMillis() - bundleDownloadStartTime) / 1000;
-                LOG.info("Bundle downloaded in {} seconds!", bundleDownloadSeconds);
-                status.update("Building graph...", 30);
             }
-            // Once bundle is downloaded, we await the build (or download if graph already built) of the graph.
-            long graphCheckStartTime = System.currentTimeMillis();
-            String graphStatusUrl = String.join("/", ipUrl, GRAPH_STATUS_FILE);
-            while (!graphIsAvailable) {
+            // Wait for otp-runner to write a status that fulfills expectations of this job
+            statusCheckStartTime = System.currentTimeMillis();
+            boolean otpRunnerCompleted = false;
+            while (!otpRunnerCompleted && !status.error) {
                 // If the request is successful, the OTP instance has started.
-                waitAndCheckInstanceHealth("graph build/download check: " + graphStatusUrl);
-                graphIsAvailable = checkForSuccessfulRequest(graphStatusUrl);
-                // wait a maximum of 4 hours if building the graph, or 20 minutes if downloading a graph
-                long maxGraphBuildOrDownloadWaitTimeMillis = graphAlreadyBuilt ? 20 * 60 * 1000 : 4 * 60 * 60 * 1000;
-                if (taskHasTimedOut(graphCheckStartTime, maxGraphBuildOrDownloadWaitTimeMillis)) {
-                    failJob("Job timed out while waiting for graph build/download. If this was a graph building machine, it may have run out of memory.");
+                waitAndCheckInstanceHealth("otp-runner completion check: " + statusUrl);
+                otpRunnerCompleted = checkForOtpRunnerCompletion(statusUrl);
+                // wait a maximum of 5 hours if building a graph, or 1 hour if just starting a server
+                long maxOtpRunnerWaitTimeMillis = (graphAlreadyBuilt ? 5 : 1) * 60 * 60 * 1000;
```

Comment:
Why did we bump up the wait time for graph build? We should justify this amount of time in the comments.

Reply:
I believe I bumped it up to 5 hours because I rounded up from the previous scheme of 20 min bundle download timeout + 4 hour graph build timeout.